src/cpp/session/SessionConsoleProcess.cpp

```diff
@@ -301,14 +306,79 @@ bool ConsoleProcess::onContinue(core::system::ProcessOperations& ops)
    return true;
 }
 
+Error ConsoleProcess::getLogFile(FilePath& file) const
+{
+   Error error = s_consoleProcPath.ensureDirectory();
+   if (error)
+   {
+      return error;
+   }
+
+   file = s_consoleProcPath.complete(handle_);
+   return Success();
+}
+
+void ConsoleProcess::deleteLogFile() const
```

Comment:
It looks like the only place this gets called is when we record the process exiting. There should be some mechanism for cleaning these up if the session crashes with console processes attached -- is that happening somewhere?